lib/annotate.rb

```diff
@@ -78,6 +78,12 @@ def self.setup_options(options = {})
     return options
   end
 
+  def self.reset_options
+    [POSITION_OPTIONS, FLAG_OPTIONS, PATH_OPTIONS, OTHER_OPTIONS].flatten.each do |key|
+      ENV[key.to_s] = nil
```

Comment:
It's needed for the tests - ENV wasn't cleared between tests, so after wrapper attributes were set in one of the tests, the others also had it set, which resulted in incorrect annotations being generated.